src/main/java/frc/robot/subsystems/arm/ArmStageOne.java

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.arm;
+
+import com.ctre.phoenix.motorcontrol.FeedbackDevice;
+import com.ctre.phoenix.motorcontrol.InvertType;
+import com.ctre.phoenix.motorcontrol.TalonSRXControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj.DutyCycleEncoder;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+import static frc.robot.Constants.ArmConstants;
+
+public class ArmStageOne extends SubsystemBase {
+  private TalonSRX armMotorPrimary;
+  private TalonSRX armMotorSecondary;
+  private DutyCycleEncoder magEncoder;
+  private double encoderOffset = ArmConstants.kStageOne_AbsEncoderInitialOffset;
+  private PIDController pid = new PIDController(ArmConstants.stageOne_kP,ArmConstants.stageOne_kI,ArmConstants.stageOne_kD);
+  private double setpointRad = encoderOffset;
+
+  /** Creates a new ArmStageOne. */
+  public ArmStageOne() {
+    armMotorPrimary = new TalonSRX(ArmConstants.kStageOne_MotorLeftID); //left is master
+    armMotorSecondary = new TalonSRX(ArmConstants.kStageOne_MotorRightID);
+
+    armMotorPrimary.configFactoryDefault();
+    armMotorSecondary.configFactoryDefault();
+
+    armMotorPrimary.follow(armMotorSecondary);
+    armMotorPrimary.setInverted(InvertType.OpposeMaster);
+
+    armMotorPrimary.configSelectedFeedbackSensor(FeedbackDevice.CTRE_MagEncoder_Relative, 0,30);
```

Comment:
I had been doing 1000ms timeouts, may want to add that to the fac default call and other calls as well.

Reply:
Okay, will do!